### PR TITLE
Update to work with ss 4.0.0-beta1

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

**Comment by tractorcow** [comment]
I've addressed an issue which made this module crash when deployed to AWS without a key and secret. Within AWS these aren't necessary, as access is controlled by permission policies.

**Comment by tractorcow** [comment]
Tested and deployed to http://ss4test-prod.sites.silverstripe.com/

File: README.md [hunk 1]
```diff
@@ -48,16 +68,16 @@ Make sure you replace `<bucket-name>` and `<your-ARN>` below with the appropriat
             "Resource": "arn:aws:s3:::<bucket-name>/public/*"
         },
         {
-            "Sid": "AllowWebsiteOnlyToProtected",
+            "Sid": "ServerFullAccess",
             "Effect": "Allow",
             "Principal": {"AWS":"<your-ARN>"},
-            "Action": "s3:GetObject",
-            "Resource": "arn:aws:s3:::<bucket-name>/protected/*"
+            "Action": "*",
```
Comment (madmatt) [inline README.md]: We shouldn't allow access to all actions, just the ones that we actually require to manage assets.

Comment (madmatt) [inline README.md]: Will fix this up after merging

**Pull request merged**